pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md

```diff
@@ -41,12 +42,12 @@ Before starting the Konvoy installation, verify that you have:
 
 1.  Create an GCP Service Principal (SP) by running the following command:
 
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
 
     <p class="message--note"><strong>NOTE: </strong>If an SP with the name exists, this command will rotate the password.</p>
 
     ```bash
-    az ad sp create-for-rbac --role contributor --name "$(whoami)-konvoy" --scopes=/subscriptions/$(az account show --query id -o tsv)
+    gcloud ad sp create-for-rbac --role contributor --name "$(whoami)-konvoy" --scopes=/subscriptions/$(az account show --query id -o tsv)
```

Comment:
FWIW, This entire command is wrong,  'ad sp ...' is the way you create a Service Account on AZURE, not GCP...

I suggest just removing the contents of all the shell excerpts in this document until you have a confirmed procedure from engineering.

FWIW, GCP service account creation looks *something like* this....

    export GCP_PROJECT=<account/project name> 
    export GOOGLE_APPLICATION_CREDENTIALS="$HOME/.gcloud/credentials.json"

    gcloud iam service-accounts create "$USER"
    gcloud projects add-iam-policy-binding $GCP_PROJECT --member="serviceAccount:$USER@$GCP_PROJECT.iam.gserviceaccount.com" --role=roles/editor
    gcloud iam service-accounts keys create $GOOGLE_APPLICATION_CREDENTIALS --iam-account="$USER@$GCP_PROJECT.iam.gserviceaccount.com"
    
    Here's the official docs on now to use the 'gcloud' command line tool for this task:   https://cloud.google.com/iam/docs/creating-managing-service-accounts#iam-service-accounts-create-gcloud


```suggestion
 <! NEED INSTRUCTIONS ON HOW TO CREATE A SP ON GCP>
```